datamap/lucene/pom.xml

```diff
@@ -141,6 +141,34 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
```

Comment:
I realize that in this pom, it should not depend on carbon-spark2, please modify the dependency in this pom

Reply:
this was added to include test suite in main CI

Reply:
Can you move the testcase to spark2 module and add `<scope>test</scope>` for spark2 in the pom